org.eclipse.scanning.api/src/org/eclipse/scanning/api/annotation/ui/FieldValue.java

```diff
@@ -135,9 +135,9 @@ public FieldDescriptor getAnnotation() {
 	}
 
 	public boolean isFileProperty() {
-		
-    	final FieldDescriptor anot = getAnnotation();
-    	if (anot!=null && anot.file()!=FileType.NONE) return true;
+
+	final FieldDescriptor anot = getAnnotation();
+	if (anot!=null && anot.file()!=FileType.NONE) return true;
 
 		Class<? extends Object> clazz;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This line will not be executed conditionally; only the first line of this 3-line block will be. The rest will execute unconditionally. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2681)